apps/storybook/stories/config.ts

```diff
@@ -28,11 +28,6 @@ const demoIndexes: DemoIndexConfig = {
       description: 'content',
       section: 'category',
     },
-    highlight: {
```

Comment:
Forgot to remove it. Thanks! You could also use highlightTitle or highlightDescription